src/storage.cc

```diff
@@ -0,0 +1,386 @@
+// storage.cc
+// Implements the Storage class
+#include "storage.h"
+
+namespace storage {
+
+// - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+Storage::Storage(cyclus::Context* ctx)
+    : cyclus::Facility(ctx) {
+  cyclus::Warn<cyclus::EXPERIMENTAL_WARNING>("the Storage is experimental.");
```

Comment:
Probably want to make it say "The Storage Facility is experimental." Also, it is worth noting that the behavior of this facility is simple enough that verification is, basically, validation, so the experimental warning may not be necessary. I think that this experimental warning may not be necessary in cases where physics need not be validated and where the facility capabilities are sufficiently covered by the tests for confidence within reason.